precode.sh

```diff
@@ -6,38 +6,43 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p task/dir1
```

Comment:
Можно и одной командой это сделать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```